### PR TITLE
Add fadein

### PR DESCRIPTION
**Pull request by kul3r4**
Solves #610 by adding a fade in effect

### CODE REVIEW

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 1]
```diff
@@ -153,6 +153,17 @@
     form#order amp-selector[name=color] div:not([selected]):not([disabled]) {
       opacity: 0.6;
     }
+    .animated {
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: I'd merge this class into the fadeIn class as these parameters are usually specific to an animation.

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 2]
```diff
@@ -403,16 +414,16 @@ <h2 class="m1">Apple</h2>
         <amp-img on="tap:AMP.setState({product: {selectedSlideForGreen: 0}})"
                   src="/img/green_apple_1_60x40.jpg"
                   width="60" height="40"
-                  class="selected"
-                  [class]="product.selectedSlideForGreen == 0 ? 'selected' : '' "
+                  class="selected animated fadeIn"
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: I wouldn't fade the elements in initially (this doesn't work for carousels anyway).

**Comment by sebastianbenz** [comment]
Looks great!

I wouldn't animate the thumbnails though as they already have a transition (elevation). The problem is that it's currently not visible because of the outline:

screen shot 2017-07-06 at 8 28 39 pm

Can you remove the outline (and the fadeIn) from these?

**Comment by kul3r4** [comment]
@sebastianbenz 

- removed the fade in from the preview
- removed the outline
- merged the css class for animation

PTAL

**Comment by sebastianbenz** [comment]
Can you merge `fadeIn` into the `show` class? Or are we using show anywhere without fade?

**Comment by kul3r4** [comment]
We use it for the preview as well, so I can't merge those two.

**Pull request merged**